frontend/src/components/Navbar.js

```diff
@@ -1,25 +1,26 @@
 import React, { useState } from 'react';
 import { Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap';
+import { Link } from "react-router-dom";
 
 function NavBar() {
   const [isOpen, setIsOpen] = useState(false);
   const toggle = () => setIsOpen(!isOpen);
   return (
     <div>
```

Comment:
This would be the Router.

Reply:
Okay

Reply:
I just realized that I left the comment on the wrong file, but you put the router in the right position. Thank you Nam.